pkg/client/restclient/config.go

```diff
@@ -158,6 +163,12 @@ func RESTClientFor(config *Config) (*RESTClient, error) {
 	if config.NegotiatedSerializer == nil {
 		return nil, fmt.Errorf("NegotiatedSerializer is required when initializing a RESTClient")
 	}
+	if config.QPS == 0.0 {
```

Comment:
This is what @lavalamp suggested. Could you two coordinate on final decision and ping again? Thanks!